### ISSUE TITLE
Dev test: Forces and Motion: Basics 2.0.0-dev.2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#579]
Forces and Motion: Basics 2.0.0-dev.2 is ready for another dev test.  Here is the dev version to be tested:
http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/2.0.0-dev.2/forces-and-motion-basics_en.html

As part of this test, please verify that the following issues have been resolved:

- [x] https://github.com/phetsims/forces-and-motion-basics/issues/187
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/184
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/183
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/182
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/180
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/178
- [x] https://github.com/phetsims/forces-and-motion-basics/issues/173

If any of these issues are not completely fixed, or you encounter any new issues, please create a new issue and reference this one.

**Comment by jessegreenberg** [comment]
It looks like that link is not the most recent version, now looking into why.

**Comment by jessegreenberg** [comment]
Redeployed, updated the link.  Dev version is correct now.

**Comment by ariel-phet** [comment]
@phet-steele @elisemorgan @brroberts1231 @BryceAG @bryo5363 please make this a high testing priority after #577

**Comment by brroberts1231** [comment]
Tested on Mac Safari, issues listed here appear to all be fixed.

**Comment by BryceAG** [comment]
Tested on Win 7, Chrome. All initial issues appear to be fixed, except for most recent comments on 187, 183, and 180.

**Comment by phet-steele** [comment]
@jessegreenberg, done.

**Comment by jessegreenberg** [comment]
Thanks @phet-steele, looks like we are almost ready for an RC!
